### PR TITLE
Update Dolphin guide/add separate pages

### PR DESCRIPTION
**Pull request by osfanbuff63**
On Windows at least, you can now just download the RiiConnect24 Patcher, and it will download the .VFF Downloader for Dolphin.

The reason I've set this up right now as two pages is I don't know how to do it on a Unix device (although I plan to test it out at some point, so that I can see if two pages is really necessary).

### CODE REVIEW

**Comment by Peter0x44** [comment]
I don't think these should be two pages. At most, one page, with two buttons for each OS, as is done on the cios page for example (the part where it explains how to change the IOS a game is running under)

**Comment by osfanbuff63** [comment]
Alright, on it!

**Comment by Peter0x44** [comment]
OK, I think it's good as far as I can tell
Merging.

**Comment by Peter0x44** [comment]
Nevermind, this doesn't work properly.
Reverting, needs more work
Screenshot_20220424_103227
Currently, there is the orange box containing no text, and also clicking the buttons has no effect.

**Comment by osfanbuff63** [comment]
I noticed, was just a bit busy. I'll work on it when I have a chance, thanks for reverting :P

File: _pages/en_US/riiconnect24-dolphin.md [hunk 1]
```diff
@@ -30,9 +30,41 @@ If you have Dolphin already installed, skip to Section II
...
 ![Perform Online System Update](/images/Dolphin_RC24/1.jpg)
 
-##### Section II - Installing RiiConnect24.
+Performing Online System Updates is [not currently possible in Dolphin](https://forums.dolphin-emu.org/Thread-perform-online-system-update?pid=524828#pid524828). You must use a [NAND dump from a real Wii](bootmii).
+{: .notice--warning}
 
-1. Run `VFF-Downloader-for-Dolphin.bat` on Windows or `VFF-Downloader-for-Dolphin.sh` on Unix systems which you downloaded [here](https://github.com/RiiConnect24/.VFF-File-Downloader-for-Dolphin/releases)
+##### Section II - Installing RiiConnect24
+
+<div class="notice--warning">{{ bruh | markdownify }}</div>
```
Comment (osfanbuff63) [inline _pages/en_US/riiconnect24-dolphin.md:38]: This would be the empty box most likely

File: _pages/en_US/riiconnect24-dolphin.md [hunk 2]
```diff
@@ -30,9 +30,41 @@ If you have Dolphin already installed, skip to Section II
...
+
+<button class="tablinks btn btn--large btn--primary" id="defaultOpen" onclick="openTab(event, 'windows')">Windows</button>
+<button class="tablinks btn btn--large btn--info" onclick="openTab(event, 'unix')">macOS/Linux</button>
```
Comment (osfanbuff63) [inline _pages/en_US/riiconnect24-dolphin.md:41]: I'll probably re-copy from the cios page and see if it has any effect.

File: _pages/en_US/riiconnect24-dolphin.md [hunk 3]
```diff
@@ -14,7 +14,7 @@ If you need help for anything regarding this tutorial, please directly contact K
 ### What you need
 * A computer with either Windows 7 or newer or any Unix based system
 * [.VFF-File-Downloader-for-Dolphin](https://github.com/RiiConnect24/.VFF-File-Downloader-for-Dolphin/releases)
```
Comment (osfanbuff63) [inline _pages/en_US/riiconnect24-dolphin.md:16]: Since this is only needed if you are using the Unix tutorial	, thinking about removing this

**Comment by Peter0x44** [comment]
> Is it possible for me to be able to deploy it so that I can test (and I tried to just enable GitHub pages and it didn't have all the formatting and such), to prevent this kind of thing?
I think so, you can run it locally using ruby and bundler

iirc the process is something like clone the repo, cd over, run `gem install bundler`, `bundle install` and `bundle exec jekyll serve`

**Pull request merged**